.editorconfig

```
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+indent_size = tab
+indent_style = tab
+tab_width = 2
```

I'm sorry but this conf is not minimal, it may break some languages

Look at the editor config file of the editorconfig project

https://github.com/editorconfig/editorconfig-core-go/blob/master/.editorconfig

These would be a better minimal version I think.

```suggestion
[*]
charset = utf-8
end_of_line = lf

[*.go]
insert_final_newline = true
indent_size = tab
indent_style = tab
tab_width = 2
```

Even if I'm unsure the tab width should be 2

---

I saw you removed the editorconfig thing

Do you plan to add it back via another PR?

---

Yes, I will add via another PR

---

PR: https://github.com/caarlos0/env/pull/327